libs/core/langchain_core/runnables/base.py

```diff
@@ -4895,17 +4895,39 @@ def with_retry(self, **kwargs: Any) -> Runnable[Input, Output]:
     def __getattr__(self, name: str) -> Any:
         attr = getattr(self.bound, name)
 
-        if callable(attr) and accepts_config(attr):
+        if callable(attr) and (
+            config_param := inspect.signature(attr).parameters.get("config")
```

Comment:
is this copy-paste logic for anywhere using `accepts_config`? Feels like this would have to extract out the config separately like this anyways.

Reply:
I think this is only useful here, the other places that check for config acceptance don't care about what kind of arg it is